README.md

```diff
@@ -1,6 +1,31 @@
-# questland-public-api
-This is a public API for Questland
+# Questland API
+This is a public API for Questland inspired by Mandalorian007. 
+Current public url: http://34.217.43.20:8080/swagger-ui.html#/
```

Comment:
https://questland-public-api.cfapps.io/swagger-ui.html

^This is the current one :)

Reply:
Looking great